client/src/app/metadata-schema-form/metadata-form/metadata-form.component.ts

```diff
@@ -54,6 +55,37 @@ export class MetadataFormComponent implements OnInit {
     this.done = true;
   }
 
+  showTab(tab: MetadataFormTab): boolean {
+    if (this.config.viewMode && this.config.removeEmptyFields && tab.items.length == 1) {
+      if (tab.key == tab.items[0]) {
+        let data = this.data;
+        let chain = true;
+
+        let keys = tab.key.replace('project.', '').split('.')
```

Comment:
`project` is hardcoded (i've made this same mistake before lol)

You could split and just shift the first element.
e.g.
```
 const keys = tab.key.split('.');
 keys.shift();

for (const key of keys)
```

Reply:
Not sure if you're familiar with command+option+l (https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat_code) to  auto reformat code (it's very handy!) It seems that this is showing some `==` vs `===` and `let` vs `const` usage issues.

Reply:
I have so many squiggles in the UI that I can't usually use that but will have a look.